recipe/meta.yaml

```
@@ -23,39 +23,31 @@ requirements:
     - setuptools
     - setuptools_scm
     - wheel
-    - cython 0.29
-    - numpy   1.19  # [py<310]
-    - numpy   1.21  # [py==310]
-    - numpy   1.23  # [py>=311]
-    - geos
+    - cython >=0.29.24
+    - numpy {{ numpy }}
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    - shapely >=1.6.4
+    - shapely >=1.7
+    - packaging >=20
     - pyshp >=2.1
-    - geos >=3.7.2
-    # including matplotlib-base even though it's optional upstream
-    - matplotlib-base >=3.1
-    - pyproj >=3.0.0
+    - matplotlib-base >=3.4
+    - pyproj >=3.1.0
   run_constrained:
     - scipy >=1.3.1
+    - pillow >=6.1.0
```

```suggestion
    - pillow >=6.1.0
    - OWSLib>=0.20.0
```
I don't think we have this package, but it doesn't hurt to add it in the `run_constained`. https://github.com/SciTools/cartopy/blob/v0.22.0/pyproject.toml#L54C9-L54C23

---

We do have `owslib 0.24.1` (the latest version)